src/modules/UI/components/FlipInput/FlipInput.ui.js

```diff
@@ -85,70 +119,116 @@ export default class FlipInput extends Component<Props, State> {
-    </View></TouchableWithoutFeedback>
+  topRow = (denominationInfo: FlipInputFieldInfo, onChangeText: ((string) => void), amount: string) => {
+    return (
+      <View style={top.row} key={'top'}>
+        <Text style={[top.symbol]}>
+          {denominationInfo.displayDenomination.symbol}
+        </Text>
+        <TextInput style={[top.amount, (Platform.OS === 'ios') ? {} : {paddingBottom: 2}]}
+          placeholder={'0'}
+          placeholderTextColor={'rgba(255, 255, 255, 0.60)'}
+          value={amount}
+          onChangeText={onChangeText}
+          autoCorrect={false}
+          keyboardType='numeric'
+          selectionColor='white'
+          returnKeyType='done'
+          underlineColorAndroid={'transparent'}
+        />
+        <Text style={[top.currencyCode]}>
+          {denominationInfo.displayDenomination.name}
+        </Text>
+      </View>
+    )
+  }
+
+  bottomRow = (denominationInfo: FlipInputFieldInfo, amount: string) => {
+    return (
+      <TouchableWithoutFeedback onPress={this.onToggleFlipInput} key={'bottom'}>
+        <View style={bottom.row}>
+          <Text style={[bottom.symbol]}>
+            {denominationInfo.displayDenomination.symbol}
+          </Text>
+          <Text style={[
+            bottom.amount,
+            !amount && bottom.alert
+          ]}>
+          {amount || '0'}
+        </Text>
+        <Text style={[bottom.currencyCode]}>
+          {denominationInfo.displayDenomination.name}
+        </Text>
+      </View>
+    </TouchableWithoutFeedback>
+    )
   }
 
-  renderRows = (primaryInfo: FlipInputFieldInfo, secondaryInfo: FlipInputFieldInfo, isToggled: boolean) => (
+  renderRows = (primaryInfo: FlipInputFieldInfo, secondaryInfo: FlipInputFieldInfo, isToggled: boolean) => {
+    return (
       <View style={[styles.rows]}>
         {isToggled
           ? [
-            this.topRow(secondaryInfo, this.onSecondaryAmountChange),
-            this.bottomRow(primaryInfo)
+            this.topRow(secondaryInfo, this.onSecondaryAmountChange, this.topDisplayAmount()),
+            this.bottomRow(primaryInfo, this.bottomDisplayAmount())
           ]
           : [
-            this.topRow(primaryInfo, this.onPrimaryAmountChange),
-            this.bottomRow(secondaryInfo)
+            this.topRow(primaryInfo, this.onPrimaryAmountChange, this.topDisplayAmount()),
+            this.bottomRow(secondaryInfo, this.bottomDisplayAmount())
           ]}
       </View>
     )
+  }
 
   render () {
     const {primaryInfo, secondaryInfo} = this.props
     const {isToggled} = this.state
-    // console.log('this.state', this.state)
-    return (
-      <View style={[styles.container]}>
-        <View style={styles.flipButton}>
-          <FAIcon style={[styles.flipIcon]} onPress={this.onToggleFlipInput} name={Constants.SWAP_VERT} size={36} />
+    const frontAnimatedStyle = {
+      transform: [
+        { rotateX: this.frontInterpolate }
+      ]
+    }
+    const backAnimatedStyle = {
```

Comment:
does this value ever change?

Reply:
Not sure how to answer, the const does not but the object within it do.

Reply:
The variable does not change the value